cmd/root.go

```diff
@@ -181,7 +181,11 @@ user created with the credentials from options "username" and "password".`,
 
 		defer listener.Close()
 
-		log.Println("Listening on", listener.Addr().String())
+		if strings.Contains(adr, "0.0.0.0") {
+			log.Println("Listening on localhost:" + server.Port)
+		} else {
+			log.Println("Listening on", listener.Addr().String())
+		}
```

Comment:
0.0.0.0 ! = localhost. This is a misleading expression and should remain as before.

Reply:
remain as 127.0.0.1 after we has change it to 0.0.0.0? But 0.0.0.0:8080 is weird.